spec/system/work_create_spec.rb

```diff
@@ -413,11 +413,11 @@
 
       # We on purpose upload a non-read me file...
       path = Rails.root.join("spec", "fixtures", "files", "orcid.csv")
-      attach_file(path) do
-        page.find("#patch_readme_file").click
+      attach_file_via_uppy(path) do
+        page.execute_script("$('#readme-upload').prop('disabled', false)")
       end
-      # ...and we expect and error message to be displayed and the button to continue to remain disabled
-      expect(page).to have_content("You must select a file that includes the word README in the name")
```

Comment:
Notice that I had to remove this test because this validation now happens within Uppy and there is no way of testing it via RSpec. 

The JavaScript for the validation is on `work_readme_file_upload.es6` in this PR, see [onBeforeLoad](https://github.com/pulibrary/pdc_describe/pull/1768/files#diff-0ce881916793c6b0eff2365f27c5cd231fc7544661eea9335cd43c66eb5642a6R28-R40)